src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -62,17 +62,17 @@ def test_csum_error_logging(self):
         host_devices = get_dmg_smd_info(dmg.storage_query_list_devices, 'devices')
         for host, devices in host_devices.items():
             for device in devices:
-                for entry in ('uuid', 'tgt_ids', 'role_bits', 'roles'):
+                for entry in ('uuid', 'tgt_ids', 'role_bits'):
```

Comment:
This change is not needed. We still want to verify the `roles` entry.
```suggestion
                for entry in ('uuid', 'tgt_ids', 'role_bits', 'roles'):
```

Reply:
I can agree with this... I can revert the change.

Reply:
I am going with Tom's change to make the test more flexible and generic.

Reply:
IMO It doesn't look like this test is intended to verify the output of the command, so I don't think we should add extraneous checks. If we do want to verify, then we should go back to check if it's `NA`.